app/views/to_do_lists/create.js.erb

```diff
@@ -1,6 +1,8 @@
 var options = {
     <% if @list.persisted? %>
         html: "<%=j render partial: 'list', locals: {list: @list} %>",
+        message: "<%=j render partial: 'create_success', locals: {page: @pager[:on_page]} %>",
+        per_page: <%= @pager[:per_page] %>,
```

Comment:
@vlasiak, it is preferable, that reference to be based on an interface of the `pager` object rather than on a hash:
```ruby
@pager.on_page # vs @pager[:on_page]
@page.per_page # vs @pager[:per_page]
```